test/controllers/ConfirmEmailControllerSpec.scala

```diff
@@ -72,7 +72,7 @@ class ConfirmEmailControllerSpec extends ControllerBaseSpec with MockVatSubscrip
 
         mockIndividualAuthorised()
 
-        val request = testGetRequest.withSession(SessionKeys.vatNumberKey -> testVatNumber, SessionKeys.emailKey -> testEmail)
+        val request = testGetRequest.withSession(SessionKeys.emailKey -> testEmail)
```

Comment:
The val `testGetRequest` in this file seems unnecessary as we already have an empty FakeRequest in `TestUtil` (it's just called `request`). Could you reuse that one?